src/components/Paginate.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { Pagination, PaginationItem, PaginationLink, Collapse } from 'reactstrap';
+
+class Paginate extends React.Component {
+  constructor(props) {
+    super(props);
+  }  
+  
+  render() {
+    return (
+        <>
+        <hr/>
+        <Pagination aria-label="Table page" className="mt-3">
+          {this.props.pages.map(page =>
```

Comment:
This is a design point and not functional, but I generally try to avoid passing these kinds of non-trivial data structures down as props. Could you instead just pass a url prefix and the page count to this component and then have it do the rest?

Reply:
my understanding is that react passes everything other than primitives by reference, so there shouldn't be a cost of passing large structures as props?

Reply:
Of course - it's not a performance issue, just awkward design from my point of view